README.markdown

```diff
@@ -250,8 +252,12 @@ override func tableView(_ tableView: UITableView, numberOfRowsInSection section:
     // #warning Incomplete implementation, return the number of rows
     return Database.contacts.count
 }
-
 ```
+
+*Note on removing dead code containing entire features / functionality:* The developer should aim to create one commit containing all feature-related code that's being removed. This commit purpose is to make it easier for the developers to bring back this functionality in the future if ever needed. To make it easier to find those commits, they should have the following message:
```

Comment:
Why put this down here rather than replace the `Mark your commit name accordingly` text up with the main part of this section's description?

Reply:
Good point, updated

Reply:
awesome thanks!